README.md

```diff
@@ -1,59 +1,303 @@
-## GnuPG application: blue-app-gnupg
+# GnuPG application
 
-GnuPG application for Nano S and Nano X
+[![Ensure compliance with Ledger guidelines](https://github.com/LedgerHQ/app-openpgp/actions/workflows/guidelines_enforcer.yml/badge.svg)](https://github.com/LedgerHQ/app-openpgp/actions/workflows/guidelines_enforcer.yml)
 
-This application implements "The OpenPGP card" specification revision 3.0. This specification is available in *doc* directory and at https://g10code.com/p-card.html .
-
-The application supports:
-  - RSA with key up to 4096 bits 
-  - ECDSA with secp256k1
-  - EDDSA with Ed25519 curve 
-  - ECDH with secp256k1 and curve25519 curves
+[![Build and run functional tests using ragger through reusable workflow](https://github.com/LedgerHQ/app-openpgp/actions/workflows/build_and_functional_tests.yml/badge.svg?branch=master)](https://github.com/LedgerHQ/app-openpgp/actions/workflows/build_and_functional_tests.yml)
 
+GnuPG application for Ledger devices
 
-This release has known missing parts (see also Add-on) :
+This application implements "The OpenPGP card" specification revision 3.3.
+This specification is available in *doc* directory and at <https://g10code.com/p-card.html>.
 
-   * Ledger Blue support
-   * Seed mode ON/OFF via apdu
+The application supports:
 
+- RSA with key up to 4096 bits
+- ECDSA with secp256k1
+- EDDSA with Ed25519 curve
+- ECDH with secp256k1 and curve25519 curves
 
 ## Installation and Usage
 
-See the full doc at https://github.com/LedgerHQ/blue-app-openpgp-card/blob/master/doc/user/blue-app-openpgp-card.pdf
-
+See the full doc at <https://github.com/LedgerHQ/app-openpgp/blob/master/doc/user/app-openpgp.pdf>
 
 ## Add-on
 
 The GnuPG application implements the following addon:
-  - serial modification
-  - on screen reset
-  - 3 independent key slots
-  - seeded key generation
 
-Technical specification is available at https://github.com/LedgerHQ/blue-app-openpgp-card/blob/master/doc/developper/gpgcard3.0-addon.rst
+- Serial modification
+- On screen reset
+- 3 independent key slots (except for Nanos, where we have onle a single slot)
+- Seeded key generation
+
+Technical specification is available at <https://github.com/LedgerHQ/app-openpgp/blob/master/doc/developer/gpgcard-addon.pdf>
```

Comment:
can we use here a relative path ?

Reply:
Fixed, with both relative path for rst and full path for pdf (which cannot be opened by default from a text/mardown editor)